seuac-credentials-plugin/README.md

```diff
@@ -43,6 +43,7 @@ The plugin defines the following tasks:
 Task name | Depends on | Type | Description
 --- | --- | --- | ---
 `setCredentials`| - | - | Sets a credential. Invoke with `--key [Key of the credentials]` parameter.
+`clearCredentials`|-|-|Clears a credential. Invoke with `--key [Key of the credentials]` parameter. Invoke without `--key` parameter to clear all stored credentials.
```

Comment:
Agreed. Thats why the task requires user confirmation (which defaults to no).

See https://github.com/clboettcher/seu-as-code.plugins/commit/5dd7d26952845683d23d6f0fce62ce7e70919767#diff-2d20732c2f579d994130d9346476abf6R55